server/mvc/log.js

```diff
@@ -0,0 +1,4 @@
+module.exports.init = function(log){
```

Comment:
No. This is just an abstraction layer on top of a pre-configured `winston` module. It works out-of-the-box.

All logs (incl. this "manual" log, as the `var log` refers to) are configured in [`server/bootstrap/logs.js`](https://github.com/Starbow/website/blob/4df559086c2b5fef4a87e2b2e02199c450f60050/server/bootstrap/logs.js).

Preferably, we should never reference anything placed in `server/bootstrap` when working with logic contained within the `server/mvc` folders, i.e. controllers and models. Bootstrap is for configuring the server, exclusively.

All models receive the full `config` Object upon initialization (see [here](https://github.com/Starbow/website/blob/4df559086c2b5fef4a87e2b2e02199c450f60050/server/mvc/models.js#L5)). However, the `config` Object does not contain logic; only data.

This means the fully configured logs `access` and `error` are completely off-limits in the MVC logic, quite intentionally.

I've made a booboo and done incorrectly in [`server/mvc/controllers/ProfileController.js#L1`](https://github.com/Starbow/website/blob/4df559086c2b5fef4a87e2b2e02199c450f60050/server/mvc/controllers/ProfileController.js#L1); the controller should have no awareness or access to configuration data, except through models. I'm going to fix this and move the logic to a model.

**TL;DR**
It's intentional. The `winston` log retrieved here is fully configured and will write log info to `manual.log`.